tonic/src/status.rs

```diff
@@ -356,7 +355,13 @@ impl Status {
             _ => Code::Unknown,
         };
 
-        Status::new(code, format!("h2 protocol error: {}", err))
+        let mut status = Self::new(code, format!("h2 protocol error: {}", err));
+        let error: Option<Box<dyn Error + Send + Sync + 'static>> = err
```

Comment:
why do you need this extra type hint?

Reply:
I don't recall. I removed it and it still compiles so pushed the removal of it.